handlers/validate/steplib/steplib.go

```diff
@@ -0,0 +1,116 @@
+package steplib
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/bitrise-io/go-utils/fileutil"
+	"github.com/bitrise-io/go-utils/pathutil"
+	version "github.com/hashicorp/go-version"
+)
+
+type StepLib struct {
```

Comment:
Why do not you use the stepman models? both the stepli, step and steplib parsing is already implemented there

Reply:
No significant difference between the this and the stepman model, however I needed the step as an array rather than a hash. (Using ID in the key and the struct field Id as well would be duplicate)